### ISSUE TITLE
JsonFluentAssert#isAbsent is not honoring TREATING_NULL_AS_ABSENT

### ISSUE THREAD

**Issue by jama707** [lukas-krecan/JsonUnit#43]
Hello, 
I have a JSON which doesn't have root node and somehow `TREATING_NULL_AS_ABSENT` is not giving any affect to the result, below is the code:
```java 
String json = "{\"a\":1, \"b\": null}";
JsonFluentAssert jsonAssert = JsonFluentAssert.assertThatJson(JsonUtils.convertToJson(json, "actual"));
jsonAssert.when(Option.TREATING_NULL_AS_ABSENT).node("b").isAbsent();
```
What could be wrong here?

Thanks!

**Comment by lukas-krecan** [comment]
Thanks for feedback. Will fix it, I have not realized that someone might want to use such combination

**Comment by lukas-krecan** [comment]
Just released version 1.16.0 with the fix
